crates/typed-store/src/rocks.rs

```diff
@@ -954,6 +954,34 @@ impl<K, V> DBMap<K, V> {
                     .db_options
                     .get_ticker_count(Ticker::BloomFilterPrefixTruePositive) as i64,
             );
+        db_metrics
+            .op_metrics
+            .rocksdb_compaction_num_bytes_written
+            .with_label_values(&[&rocksdb.db_name()])
```

Comment:
question: should we use cf_name instead here? compaction is on cf level, correct?

Reply:
Statistics is per db in rocksdb unfortunately (per cf is not supported)